src/Config/MutationsConfig.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
I'd name it in other way as I understand this class is a collection of `MutationConfig`

Reply:
I don't see a better name 😬